src/alert-prelude.c

```diff
@@ -207,12 +207,18 @@ static int EventToImpact(const PacketAlert *pa, const Packet *p, idmef_alert_t *
     SCEnter();
 
     ret = idmef_alert_new_assessment(alert, &assessment);
-    if (unlikely(ret < 0))
+    if (unlikely(ret < 0)) {
+        SCLogDebug("%s: error creating assessment: %s.",
```

Comment:
If it is really an error then we should use a SCLogError but we don't want to flood logs if it happens for every alerts.